src/config/audioLinks.ts

```diff
@@ -0,0 +1,4 @@
+export enum AudioLinks {
+  Cabeza = "https://pub-27907ac1752340b79f24b92b358d3644.r2.dev/la_cabeza.mp3",
+  CabezaSentence = "https://pub-27907ac1752340b79f24b92b358d3644.r2.dev/la_cabeza_sentence.mp3"
+}
```

Comment:
Awesome, but you could also extract server URL to a separate variable.

```suggestion
const AUDIO_SERVER_URL = "https://pub-27907ac1752340b79f24b92b358d3644.r2.dev";

export enum AudioLinks {
  Cabeza = `${AUDIO_SERVER_URL}/la_cabeza.mp3`,
  CabezaSentence = `${AUDIO_SERVER_URL}/la_cabeza_sentence.mp3`
}
```